### PR TITLE
Add swig support for nodejs

### PR DESCRIPTION
**Pull request by 599316527**

### CODE REVIEW

File: src/swig/Makefile [hunk 1]
```diff
@@ -6,10 +6,10 @@ all clean:
 	for subdir in $$list; do \
 		if [ -x $$subdir/build -a ! -f .$$subdir -o $@ = clean ] ; \
 		then echo -n Building $$subdir... ; \
-			cd $$subdir && output=$$(CXXFLAGS="$(CXXFLAGS)" ./build $@ 2>&1) ; \
```
Comment (ddennedy) [inline src/swig/Makefile]: Great contribution! But why this change?

Comment (599316527) [reply src/swig/Makefile]: Putting output into a variable and echoing the variable works weird in my mac. Line-breaks are missing.

compare

And I believe that build shell's output which might be very important and helpful for those who build their own libraries to find out which step goes wrong or warning shall be always shown.

Comment (ddennedy) [reply src/swig/Makefile]: Same on Linux, and I noticed some useful warnings (e.g. operator=) while testing this change even when successful.
